### PR TITLE
Flake8 fix

### PR DESCRIPTION
**Pull request by makaramkd**
Flake8 issues are fixed. Ive just added `A003` and `C901` as ignore params  in `.flake8` because `A003` is just showing the shadow variables in the models like `id` and `C901` is just showing complexity of a function in `v1/tasks/clean.py`.

### CODE REVIEW

**Review by vosi — CHANGES_REQUESTED** [review]

File: .flake8 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = D203,D101,D400,D401,D106,W503,F403,F405,E501,D403,D104,C101,D100,D105,D103,D102
+ignore = D203,D101,D400,D401,D106,W503,F403,F405,E501,D403,D104,C101,D100,D105,D103,D102,A003,C901
```
[the remark this answers is not in the document]
Comment (makaramkd) [reply .flake8:2]: What do you mean by old code? For now I just used the `#noqa` notation on the attributes that are shadowing the python builtin, there were like 3 of them. For example:
`v1/self_configurations/management/commands/initialize_test_bank.py#32`:  
`help = 'Delete existing data, load in fixture data, and rebuild cache'  # noqa: A003`

[the remark this answers is not in the document]
Comment (vosi) [reply .flake8:2]: in that case, remove your addition to ignore

**Comment by makaramkd** [comment]
@vosi All the requested changes are done.

**Comment by vosi** [comment]
@makaramkd great, thanks, the only thing left is to ensure that Checks pass

**Comment by makaramkd** [comment]
> @makaramkd great, thanks, the only thing left is to ensure that Checks pass

I am a bit confused about the checks. Its showing error in file: 
`./docs/source/conf.py:23:1`, `A001 variable "copyright" is shadowing a python builtin` as I think its a `sphinx` documentation . But I cannot find the file in the project. Also on my local machine `flake8 .` passes 100%.

**Comment by vosi** [comment]
@makaramkd that's because upstream was updated, and github actions do an internal merge to test your branch as it is merged. You can update or rebase your branch with upstream

**Comment by makaramkd** [comment]
@vosi Its done. I forgot to check the upstream before. :)

**Review by vosi — APPROVED** [review]

**Comment by makaramkd** [comment]
My account number: 
`b0302a7a0c02df1a5d298122de7c22a0bdd2394de75d1ae724f51a85bd13128e`

**Comment by vosi** [comment]
@makaramkd I dont do payouts, ask someoune from https://thenewboston.com/teams Payouts team

**Comment by makaramkd** [comment]
Reference: Fix Flake8 Issues for Bank #96

**Pull request merged**